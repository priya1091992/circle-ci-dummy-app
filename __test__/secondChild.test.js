import React from 'react';
import SecondChild from '../src/app/secondChild.js';
import renderer from 'react-test-renderer';

var list = ['a', 'b', 'c']
test('render list', () => {
  const wrapper = renderer.create(
      <SecondChild list={list}/>,
  );
  let tree = wrapper.toJSON();

  expect(tree).toMatchSnapshot();
});
