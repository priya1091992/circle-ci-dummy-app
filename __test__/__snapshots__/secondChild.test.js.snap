// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render list 1`] = `
<div
  className="container"
>
  <hr />
  <h3>
    a
  </h3>
  <h3>
    b
  </h3>
  <h3>
    c
  </h3>
</div>
`;
